prow/new-e2e-rbac_no_auth.sh

```diff
@@ -54,8 +52,18 @@ gcloud container clusters create ${CLUSTER_NAME} --zone ${ZONE} --project ${PROJ
   || { echo "Failed to create a new cluster"; exit 1; }
 CLUSTER_CREATED=true
 
+for i in {1..10}
+do
+  kubectl get namespace
+  if [ ${?} -eq 0 ]; then
+    break
+  fi
+  if [ ${i} -eq 10 ]; then
+    echo "Cannot connect to the new cluster"; exit 1
+  fi
+  sleep 5
+done
+
 kubectl create clusterrolebinding prow-cluster-admin-binding --clusterrole=cluster-admin --user=istio-prow-test-job@istio-testing.iam.gserviceaccount.com
 
-echo 'Running e2e rbac, no auth Tests'
 ./prow/e2e-suite-rbac-no_auth.sh "${@}"
```

Comment:
without the set -e don't you need 
exec ./prow/e2e-suite-rbac-no_auth.sh "${@}"
to carry the exit code up ?
(it's better anyway for the last line of any script to be exec nextscript even if the exit isn't an issue)

Reply:
It seems not necessary but right, it's good to exit with explicit code.